### ISSUE TITLE
lt-ie9 styles get applied in Firefox and Chrome

### ISSUE THREAD

**Issue by Neo-Oli** [thingles/foreground#310]
### Setup

- Browsers and versions: Firefox 57.0.2 and Chrome 63.0.3239.84
- MediaWiki version: 1.30.0
- Database version: 10.1.29 MariaDB
- PHP version: 7.0.22
- Foreground version: master

### Issue

The `lt-ie9` class gets added to the `<html>` element. This only started happening after the upgrade from 1.29.2 to 1.30.0. This causes the header bar to  split into 3 lines.

**Comment by Hutchy68** [comment]
Time to discuss supporting IE 8, but here is the temp fix for all modern browsers.

Find line # 4 in `/assets/scripts/foreground.js` and comment out this

```
  if (!jQuery.support.cssFloat) { jQuery('html').addClass('lt-ie9').addClass('no-js'); }
```

```
  // if (!jQuery.support.cssFloat) { jQuery('html').addClass('lt-ie9').addClass('no-js'); }
```

**Comment by kghbln** [comment]
Since MW is no longer supporting [IE 8](https://www.mediawiki.org/wiki/Compatibility#Browser_support_matrix) either this fix should imho go into 2.0

**Comment by Hutchy68** [comment]
Support was dropped in 2.1.0-rc, closing this as fixed
